.github/workflows/ios.yml

```diff
@@ -0,0 +1,37 @@
+name: iOS CI
+
+on:
+  push:
+    branches: [ master]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build and Test default scheme using any available iPhone simulator
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Relisten
+        run: |
+          git clone --quiet https://github.com/alecgorge/gapless-audio-bass-ios.git ../BASSGaplessAudioPlayer
+          git clone --quiet https://github.com/alecgorge/AGAudioPlayer.git ../AGAudioPlayer
+          pod update RealmSwift
+          ./setup.sh
+          scheme_list=$(xcodebuild -list -json | tr -d "\n")
```

Comment:
We practically never change the scheme name, so I think it's safer to hardcode it here than to rely on scripts to parse it out.